src/lib/moneyHelpers.ts

```diff
@@ -39,49 +39,46 @@ export const isCurrencySupported = (currency: string): boolean => {
 }
 
 /**
- * Builds price display text (e.g. "$100").
+ * Builds price display text (e.g. "US$100").
  */
 export const priceDisplayText = (
-  priceCents: number | [number, number],
```

Comment:
Reverting this change from https://github.com/artsy/metaphysics/pull/3393 to make the responsibility clear: formatting a single number.